command/results_utils.go

```diff
@@ -17,55 +17,57 @@ import (
 )
 
 // sendTestResults sends the test results to the API server and Cedar.
-func sendTestResults(ctx context.Context, conf *model.TaskConfig,
-	logger client.LoggerProducer, comm client.Communicator,
-	results *task.LocalTestResults) error {
-
+func sendTestResults(ctx context.Context, comm client.Communicator, logger client.LoggerProducer, conf *model.TaskConfig, results *task.LocalTestResults) error {
 	if results == nil || len(results.Results) == 0 {
 		return errors.New("cannot send nil results")
 	}
 
-	logger.Execution().Info("attaching test results")
-
+	logger.Task().Info("Attaching results to server...")
 	td := client.TaskData{ID: conf.Task.Id, Secret: conf.Task.Secret}
-	if err := comm.SendTestResults(ctx, td, results); err != nil {
-		return errors.WithStack(err)
+	if conf.ProjectRef.CedarTestResultsEnabled {
+		if err := sendTestResultsToCedar(ctx, conf.Task, td, comm, results); err != nil {
+			logger.Task().Errorf("problem posting parsed results to the cedar: %+v", err)
+			return errors.Wrap(err, "problem sending test results to cedar")
+		}
+	} else {
+		if err := comm.SendTestResults(ctx, td, results); err != nil {
+			logger.Task().Errorf("problem posting parsed results to evergreen: %+v", err)
+			return errors.Wrap(err, "problem sending test results to evergreen")
+		}
 	}
+	logger.Task().Info("Successfully attached results to server")
 
-	// TODO (EVG-7780): send test results for projects that enable it.
-	// if err := sendTestResultsToCedar(ctx, conf.Task, td, comm, results); err != nil {
-	//     return errors.Wrap(err, "sending test results to Cedar")
-	// }
+	return nil
+}
 
-	logger.Task().Info("Attach test results succeeded")
+// sendTestLog sends test logs to the API server and Cedar.
+func sendTestLog(ctx context.Context, comm client.Communicator, conf *model.TaskConfig, log *model.TestLog) (string, error) {
+	if conf.ProjectRef.CedarTestResultsEnabled {
+		return "", errors.Wrap(sendTestLogToCedar(ctx, conf.Task, comm, log), "problem sending test logs to cedar")
```

Comment:
Can we have this send to Cedar to both Evergreen and Cedar until we fully enable sending all test logs to Cedar? Just in case we test it on projects, we still want to be able to retrieve the logs in case we need to turn the project setting off.

Reply:
I think it would be better to send it to the one or the other, instead of duplicating logs. We can turn it on slowly (first one of our projects, then a few of them, then all of them, and finally all projects in evergreen). If something goes wrong we can always switch back to sending test results to evg.

Reply:
My problem isn't with switching back to sending them to evergreen, it's that logs that were already sent to only Cedar but not Evergreen will not be viewable from the UI if you turn off sending test results to the Evergreen apps. This is only supposed to be a stopgap and test logs aren't supposed to be huge. I would prefer users have their logs available at all times if they turn off the project setting to get their logs from Evergreen.

Reply:
I would argue that we shouldn't turn it on for anything until the UI work is done (which is my next ticket). But I think you feel more strongly about this than I do so I will change it.

Reply:
Like the UI code for buildlogger always tries cedar first and then falls back to evergreen. So even if we change things, it checks both.

Reply:
Hmm I guess I was assuming we wouldn't check both endpoints from the UI, but I'm fine with this if it tries both.